src/components/business/recordpage/audioRecord.tsx

```diff
@@ -0,0 +1,101 @@
+'use client'
```

Comment:
Do you use this file? Wrong way to set name of folders, files. Please check it carefully